### PR TITLE
added mark up to component AddDrinkForm

### PR DESCRIPTION
**Pull request by sgrischenko8**

### CODE REVIEW

**Comment by DanyloTytarenko** [comment]
Use styled components instead of inline styles please

File: src/components/AddDrinkForm/DrinkDescriptionFields/DrinkDescriptionFields.jsx [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+          title="Enter item title"
+          style={{
+            paddingBottom: '14px',
+            backgroundColor: 'inherit',
+            color: '#f3f3f3',
+            letterSpacing: '-0.02em',
+            outline: 'none',
+            border: 'none',
+            borderBottom: '1px solid rgba(243, 243, 243, 0.5)',
+            textTransform: 'capitalize',
```
Comment (DanyloTytarenko) [inline src/components/AddDrinkForm/DrinkDescriptionFields/DrinkDescriptionFields.jsx]: add handler to the onChange event

File: src/components/AddDrinkForm/DrinkDescriptionFields/DrinkDescriptionFields.jsx [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+          title="Enter about recipe"
+          style={{
+            paddingBottom: '14px',
+            backgroundColor: 'inherit',
+            color: '#f3f3f3',
+            letterSpacing: '-0.02em',
+            outline: 'none',
+            border: 'none',
+            borderBottom: '1px solid rgba(243, 243, 243, 0.5)',
+            textTransform: 'capitalize',
```
Comment (DanyloTytarenko) [inline src/components/AddDrinkForm/DrinkDescriptionFields/DrinkDescriptionFields.jsx]: Same as above

**Review by DanyloTytarenko — CHANGES_REQUESTED** [review]

File: src/components/AddDrinkForm/DrinkDescriptionFields/DrinkDescriptionFields.jsx (hunk 1 above)
Comment (sgrischenko8) [reply src/components/AddDrinkForm/DrinkDescriptionFields/DrinkDescriptionFields.jsx]: added onChange function

File: src/components/AddDrinkForm/DrinkDescriptionFields/DrinkDescriptionFields.jsx (hunk 2 above)
Comment (sgrischenko8) [reply src/components/AddDrinkForm/DrinkDescriptionFields/DrinkDescriptionFields.jsx]: added onChange function

**Review by DanyloTytarenko — APPROVED** [review]

**Pull request merged**